fvtest/porttest/CMakeLists.txt

```diff
@@ -75,30 +45,60 @@ omr_add_executable(omrporttest
 	si_numcpusTest.cpp
 	testHelpers.cpp
 	testProcessHelpers.cpp
+)
 
-	${cudaSources}
-	${socketSources}
+target_include_directories(omrporttest
+	PRIVATE
+		${CMAKE_CURRENT_SOURCE_DIR}
```

Comment:
This is a really minor, personal choice, but I like to add all the sources first, _then_ add include directories and link libraries.

Reply:
Sure! I can re arrange the files

Reply:
I also rearranged the `sltestlib` to be in the same section of the file, and in the same order as `aixbaddep`, for the add library, setting property and export calls.